src/api/app/views/layouts/webui2/webui.html.haml

```diff
@@ -53,7 +53,12 @@
     .modal.fade#modal{ tabindex: '-1', role: 'dialog', aria: { labelledby: 'modalLabel', hidden: true } }
 
     .container.flex-grow#content
-      = yield
+      - if controller_name == 'apidocs'
```

Comment:
Phew I don't really like that. Can we at least record an issue that we want to fix this in the future?

Reply:
@hennevogel I've developed a different approach. Could you have another look?